test/builder-api/src/packages.js

```diff
@@ -230,19 +230,6 @@ describe('Working with packages', function () {
         });
     });
 
-    it('upload does not allow native packages until feature is set', function (done) {
```

Comment:
Instead of removing this test, modify it to ensure that native package upload works.